files/logstash-configs/23-ufw.conf

```diff
@@ -15,4 +15,17 @@ filter {
       }
     }
   }
+  if [type] == "ufw" {
+    grok {
+      match => { "message" => "\[%{DATA}\] \[UFW %{WORD:ufw_action}\] IN=%{DATA:ufw_interface} OUT= MAC=%{DATA:ufw_mac} SRC=%{IP:ufw_src_ip} DST=%{IP:ufw_dest_ip} %{GREEDYDATA:ufw_tcp_opts} PROTO=%{WORD:ufw_protocol} SPT=%{INT:ufw_src_port} DPT=%{INT:ufw_dst_port} %{GREEDYDATA:ufw_tcp_opts}" }
+    }
+    if [ufw_src_ip] {
```

Comment:
The namespaced vars (with prefix `ufw_`) are a good addition, but I don't like the repetition of the geoip block. If you change the `ufw_src_ip` -> `src_ip`, or vice versa, then a single geoip block in its own if statement at the bottom of the file would catch all examples. More DRY that way!